<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TocDlg</class>
 <widget class="QDialog" name="TocDlg">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>537</width>
    <height>385</height>
   </rect>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Table of Contents</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTreeWidget" name="treeWidget">
     <property name="minimumSize">
      <size>
       <width>519</width>
       <height>0</height>
      </size>
     </property>
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="autoScroll">
      <bool>false</bool>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="uniformRowHeights">
      <bool>true</bool>
     </property>
     <property name="itemsExpandable">
      <bool>true</bool>
     </property>
     <property name="allColumnsShowFocus">
      <bool>false</bool>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <property name="headerHidden">
      <bool>true</bool>
     </property>
     <property name="expandsOnDoubleClick">
      <bool>false</bool>
     </property>
     <attribute name="headerVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="headerStretchLastSection">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string notr="true">1</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Type page number:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="pageNumEdit">
       <property name="focusPolicy">
        <enum>Qt::ClickFocus</enum>
       </property>
       <property name="maxLength">
        <number>9</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <action name="actionGotoPage">
   <property name="text">
    <string notr="true">GotoPage</string>
   </property>
   <property name="toolTip">
    <string notr="true">GotoPage</string>
   </property>
   <property name="shortcut">
    <string notr="true">Clear</string>
   </property>
  </action>
  <action name="actionNextPage">
   <property name="text">
    <string notr="true">NextPage</string>
   </property>
   <property name="toolTip">
    <string notr="true">NextPage</string>
   </property>
   <property name="shortcut">
    <string notr="true">PgDown</string>
   </property>
  </action>
  <action name="actionPrevPage">
   <property name="text">
    <string notr="true">PrevPage</string>
   </property>
   <property name="toolTip">
    <string notr="true">PrevPage</string>
   </property>
   <property name="shortcut">
    <string notr="true">PgUp</string>
   </property>
  </action>
  <action name="actionUpdatePage">
   <property name="text">
    <string notr="true">UpdatePage</string>
   </property>
   <property name="toolTip">
    <string notr="true">UpdatePage</string>
   </property>
   <property name="shortcut">
    <string notr="true"/>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>treeWidget</sender>
   <signal>itemExpanded(QTreeWidgetItem*)</signal>
   <receiver>actionUpdatePage</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>268</x>
     <y>174</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>treeWidget</sender>
   <signal>itemCollapsed(QTreeWidgetItem*)</signal>
   <receiver>actionUpdatePage</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>268</x>
     <y>174</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
